demo/src/com/google/maps/android/utils/demo/KmlDemoActivity.java

```diff
@@ -0,0 +1,92 @@
+package com.google.maps.android.utils.demo;
+
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.maps.android.kml.KmlContainer;
+import com.google.maps.android.kml.KmlLayer;
+import com.google.maps.android.kml.KmlPlacemark;
+import com.google.maps.android.kml.KmlPolygon;
+
+import org.xmlpull.v1.XmlPullParserException;
+
+import android.graphics.Typeface;
+import android.os.AsyncTask;
+import android.text.Html;
+import android.text.method.ScrollingMovementMethod;
+import android.util.Log;
+import android.widget.TextView;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+public class KmlDemoActivity extends BaseDemoActivity {
+    GoogleMap mMap;
+
+    protected int getLayoutId() {
+        return R.layout.kml_demo;
+    }
+
+    public void startDemo () {
+        try {
+            Log.i("Demo", "Start");
```

Comment:
Are these logs useful for anything? They look like leftover debug statements. If they're useful, make the messages more descriptive, if they're not, remove them.